pkg/config/config.go

```diff
@@ -47,16 +47,31 @@ var c config
 var (
 	EnabledEBPFCgroupID = false
 
-	EstimatorModel        = "" // auto-select
-	EstimatorSelectFilter = "" // no filter
-	CoreUsageMetric       = "curr_cpu_cycles"
-	DRAMUsageMetric       = "curr_cache_miss"
-	UncoreUsageMetric     = ""                // no metric (evenly divided)
-	GeneralUsageMetric    = "curr_cpu_cycles" // for uncategorized energy; pkg - core - uncore
+	EstimatorModel        = getConfig("EstimatorModel", "")        // auto-select
+	EstimatorSelectFilter = getConfig("EstimatorSelectFilter", "") // no filter
+	CoreUsageMetric       = getConfig("CoreUsageMetric", "cpu_instr")
+	DRAMUsageMetric       = getConfig("DRAMUsageMetric", "cache_miss")
+	UncoreUsageMetric     = getConfig("UncoreUsageMetric", "")           // no metric (evenly divided)
+	GeneralUsageMetric    = getConfig("GeneralUsageMetric", "cpu_instr") // for uncategorized energy; pkg - core - uncore
 
 	versionRegex = regexp.MustCompile(`^(\d+)\.(\d+).`)
+
+	ConfigPath = "/etc/config"
 )
 
+func getConfig(key, defaultValue string) string {
+	configFile := fmt.Sprintf("%s/%s", ConfigPath, key)
+	value, err := os.ReadFile(configFile)
+	if err == nil {
+		return string(value)
+	}
+	strValue, present := os.LookupEnv(key)
```

Comment:
is this a best practice that use e.g `DRAMUsageMetric` as ENV VAR instead of something like `DRAM_USAGE_METRIC` ?

Reply:
I also think that env variables should be `DRAM_USAGE_METRIC`

Reply:
I also think that all-capital should be more common for environment variable. 
I just change it to the same format as go variables for simplicity as golint asks me to change the name to camel case. 
Anyway, I will change key string back to all-capital case.

Reply:
Made an update (to all-capital case) here.

https://github.com/sustainable-computing-io/kepler/pull/284/commits/0c113ad1be69cc6422f419e6ff8c085ebdbdb7e3#diff-a3d824da3c42420cd5cbb0a4a2c0e7b5bfddd819652788a0596d195dc6e31fa5